app/views/content/subjects/physics/_article.html.erb

```diff
@@ -148,6 +148,11 @@
     <p><a href="/funding-and-support/scholarships-and-bursaries">Find out about your eligibility for scholarships and bursaries</a>.</p>
     <p>You can <a href="https://www.iop.org/about/support-grants/iop-teacher-training-scholarships">find out more about the physics scholarship from the Institute of Physics</a>.</p>
   </section>
+  <section class="col col-space-m col-720">
+  <h3>Improving your physics subject knowledge</h3>
```

Comment:
```suggestion
  <h3>Improve your physics subject knowledge</h3>
```

Suggest changing this for consistency with other h3s on the page (I appreciate the ones on the maths page are different!)